### PR TITLE
manual orb auth steps

### PR DESCRIPTION
**Pull request by rosieyohannan**
* retain manual orb creation steps to link to as an alternative to the orb dev kit

### CODE REVIEW

**Comment by rosieyohannan** [comment]
To do: replace basic orb create snippet with better example

**Review by melyalex — APPROVED** [review]

**Review by melyalex — APPROVED** [review]

**Review by melyalex — APPROVED** [review]

**Review by melyalex — APPROVED** [review]

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 1]
```diff
@@ -1,107 +1,45 @@
...
-order: 1
-sitemap: false
+title: "Manual Orb Authoring Process"
+description: "Authoring simple orbs manually without the orb development kit."
 version:
 - Cloud
 ---
 
-## Introduction
+This guide covers the steps required to create a simple orb manually, without using the orb development kit. We recommend the orb development kit for most orb projects, to find out more, see the [Orb Authoring Guide]({{site.baseurl}}/2.0/orb-author).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:9]:
```suggestion
This guide covers the steps required to create a simple orb, manually, without using the orb development kit. We recommend the orb development kit for most orb projects, to find out more, see the [Orb Authoring Guide]({{site.baseurl}}/2.0/orb-author).
```

**Review by KyleTryon — APPROVED** [review]

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 2]
```diff
@@ -1,107 +1,45 @@
...
-attach-workspace | Boolean for whether or not to attach to an existing workspace. Default is false.
-workspace-root | Workspace root path that is either an absolute path or a path relative to the working directory. Defaults to ‘.’ (the working directory)
-{: class="table table-striped"}
+1. Create your orb inside your namespace. At this stage no orb content is being generated, but the naming is reserved for when the orb is published:
+```sh
+circleci orb create <my-namespace>/<my-orb-name>
+```
 
-### Validate and Publish Example
+1. Create the content of your orb in a YAML file. Generally, you would do this in your code editor, in a git repo made for your orb, but, For this simple example, let's assume a file in `/tmp/orb.yml` could be made with a bare-bones orb like:
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:22]:
```suggestion
1. Create the content of your orb in a YAML file. For example:
```yaml
version: 2.1
description: A greeting command orb
commands:
    greet:
        description: Greet someone with a "hello".
        parameters:
            to:
                type: string
                default: World
        steps:
            - run: echo "Hello, << parameters.to >>"
```
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 3]
```diff
@@ -1,107 +1,57 @@
...
+        description: Greet someone with a "hello".
+        parameters:
+            to:
+                type: string
+                default: World
+        steps:
+            - run: echo "Hello, << parameters.to >>"
+```sh
+echo 'version: "2.1"\ndescription: "a sample orb"' > /tmp/orb.yml
+```
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:37]:
```suggestion
```

File: jekyll/_cci2/orb-author-validate-publish.md [hunk 4]
```diff
@@ -1,107 +1,54 @@
...
+description: A greeting command orb
+commands:
+    greet:
+        description: Greet someone with a "hello".
+        parameters:
+            to:
+                type: string
+                default: World
+        steps:
+            - run: echo "Hello, << parameters.to >>"
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-author-validate-publish.md:34]:
```suggestion
            - run: echo "Hello, << parameters.to >>"
```
```

**Pull request merged**